### ISSUE TITLE
Eyes burning sensation

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9805]
**Preferred term label:**
Eyes burning sensation

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Blepharitis, infectious

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000420
- label: Burning eye sensation
- parents: [Ocular pain (HP:0200026)](https://hpo.jax.org/app/browse/term/HP:0200026)
- pmid: 
- def: A feeling of stinging or irritation of the eyes.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9805
- synonyms:
